src/Components/Web/src/Forms/InputFile/IBrowserFile.cs

```diff
@@ -38,7 +38,20 @@ public interface IBrowserFile
         /// <summary>
         /// Opens the stream for reading the uploaded file.
         /// </summary>
+        /// <param name="maxAllowedSize">
+        /// The maximum size that can be read from the Stream. Defaults to 500kb.
+        /// <para>
+        /// Calling <see cref="OpenReadStream(long, CancellationToken)"/>
+        /// will throw if the uploaded file size, as specified by <see cref="Size"/> is larger than
+        /// <paramref name="maxAllowedSize"/>. By default, uploading a file larger than 500kb will result in an exception.
```

Comment:
```suggestion
        /// <paramref name="maxAllowedSize"/>. By default, if the user supplies a file larger than 500 KB, this method will throw an exception.
```

Reply:
Switching to more active voice to clarify who are the actors that supply files and throw exceptions.